### ISSUE TITLE
Wrapped links in strackstrace not clickable

### ISSUE THREAD

**Issue by crstnbr** [JunoLab/Juno.jl#122]
I hope this is the right place to post this.

I observe that stacktrace links to a line in the source code in the Juno console aren't clickable when they are wrapped to a new line, as for example due to a too small console. If the console is wide enough, they are clickable.

**Comment by pfitzseb** [comment]
Yeah, this is a upstream issue. As soon as the xterm.js supports multiline links we will too.
